lib/src/ovis_json/ovis_json.c

```diff
@@ -654,6 +668,12 @@ json_str_t json_value_str(json_entity_t value)
 	return value->value.str_;
 }
 
+const char * json_value_str_str(json_entity_t value)
```

Comment:
I'm not sure I see the value of this interface. A json_str_t is a counted string and this interface hides that.

Reply:
The value in this is that we stop writing instances of

value->value.str_->str

any place we need to print the entity string value or compare the entity string value to another non-json string.
Why would we want to keep breaking the entity encapsulation specifically for strings when we have similar operators for json_value_int, etc?

Reply:
You're breaking the entity encapsulation, not preserving it. There is already a json_value_str(e), the counted string is accessed as json_value_str(e)->str and json_value_str(e)->len.

Reply:
Why would I want to be writing 
```
json_value_str(e)->str
instead of
json_value_str_str(e)
```
Seems like every other string in the c/c++ world provides some form of "just give me the pointer".

Actually, what I'd really like to see is
const char *json_value_str(e);
and
struct json_str_s *json_value_jstr(e);
since I find only a single instance of json_value_str in the ldms code which is not used as json_value_str(e)->str (in calling papi_process_config_data) and 144 instances of jvs(e)->str.

If that's an acceptable change, I'll volunteer to make it.